packages/dd-trace/src/tracer.js

```diff
@@ -122,6 +122,33 @@ class DatadogTracer extends Tracer {
 <meta name="dd-trace-id" content="${traceId}" />\
 <meta name="dd-trace-time" content="${traceTime}" />`
   }
+
+  /**
+   * Links an authenticated user to the current trace
+   * @param {object} user Properties of the authenticated user. Accepts custom fields
+   * @param {string} user.id Unique identifier of the user. Mandatory
+   * @param {string} [user.email] Email of the user
+   * @param {string} [user.name] User-friendly name of the user
+   * @param {string} [user.session_id] Session ID of the user
+   * @param {string} [user.role]
+   * @param {string} [user.scope]
+   * @returns {boolean} If the operation succeeded
+   */
+  setUser (user) {
+    if (!user || !user.id) return false
+
+    const span = this.scope().active()
+    if (!span) return false
+
+    const rootSpan = span._spanContext._trace.started[0]
+    if (!rootSpan) return false
+
+    for (const k of Object.keys(user)) {
+      rootSpan.setTag(`usr.${k}`, '' + user[k])
```

Comment:
Are these tags shown in the trace UI or are they removed? If they are removed we should not add additional tags.

Reply:
They're shown!